www/src/templates/template-creator-details.js

```diff
@@ -40,8 +40,8 @@ const MetaSection = ({ children, background, last, first }) => (
   <div
     sx={{
       background: background ? background : `ui.background`,
-      mx: t => `-${t.space[5]}`,
-      p: 5,
+      mx: -6,
```

Comment:
These two look different? Was that something you were looking at and changed as a preference I take it? Probably not a big deal.

Reply:
Thanks for being thorough! 🤗
That's 9963afc and that was intentionally! ✌️

Reply:
> From my understanding, adding these sx theme aware properties makes these values codemod-able so we can add or remove from the spacing scale easier.

That, and to stick to using the `theme-ui` ["theme-aware properties" sauce](https://theme-ui.com/sx-prop/#theme-aware-properties) consistently.